src/Resources/ResourceManager/Implementation/TemplateSpecs/SetAzTemplateSpec.cs

```diff
@@ -138,9 +139,9 @@ public class SetAzTemplateSpec : TemplateSpecCmdletBase
 
         [Alias("InputFile")]
         [Parameter(Mandatory = true, ParameterSetName = UpdateVersionByNameFromJsonFileParameterSet, ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The file path to the local Azure Resource Manager template JSON file.")]
+            HelpMessage = "The file path to the local Azure Resource Manager template JSON/Bicep file.")]
```

Comment:
You want to update this line too https://github.com/Azure/azure-powershell/pull/15397/files#diff-6303f09498ab2990faf33994596e3d19ce6e45fa9056372e7bc874faa2c61dd6R123